pkg/controller/route/routecontroller.go

```diff
@@ -79,7 +84,12 @@ func (rc *RouteController) reconcile(nodes []api.Node, routes []*cloudprovider.R
 	for _, route := range routes {
 		routeMap[route.TargetInstance] = route
 	}
+
+	maxRetries := 5
+	maxConcurrentRouteCreations := 200
```

Comment:
Nit: Can you move this magic number to the top? I care less about the retry count, but for that one you can say:
`const maxretries = 5` (or just embed it in the loop)